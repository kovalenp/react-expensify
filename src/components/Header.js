import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <div>
    <header>
      Expensify
    </header>
      <NavLink activeClassName='selected-menu' exact to='/'>Home</NavLink>
      <NavLink activeClassName='selected-menu' to='/create'>Add Expense</NavLink>
      <NavLink activeClassName='selected-menu' to='/help'>Help</NavLink>
  </div>
);

export default Header;
