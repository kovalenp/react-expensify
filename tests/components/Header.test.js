import React from 'react';
import ReactShallowRednderer from 'react-test-renderer/shallow';
import Header from '../../src/components/Header'

test('should render Header correctly', () => {
  const renderer = new ReactShallowRednderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
