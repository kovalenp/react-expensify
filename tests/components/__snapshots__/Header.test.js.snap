// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <header>
    Expensify App
  </header>
  <NavLink
    activeClassName="selected-menu"
    aria-current="page"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="selected-menu"
    aria-current="page"
    to="/create"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="selected-menu"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
